.github/workflows/ci.yml

```
@@ -10,8 +10,8 @@ jobs:
       max-parallel: 10
       fail-fast: false
       matrix:
-        php: ['8.0', '8.1', '8.2']
-        sf_version: ['5.2.*', '5.4.*', '6.0.*']
+        php: ['8.0', '8.1', '8.2', '8.3']
+        sf_version: ['5.4.*', '6.4.*', '7.0.*']
```

The matrix probably needs to be changed as followed:
```suggestion
        php: ['8.2', '8.3']
        sf_version: ['7.0.*']
        include:
            - php: '8.0'
              sf_version: '5.4.*'
            - php: '8.1'
              sf_version: '6.4.*'
```
Because the Symfony versions have a different min PHP requirement:
| Symfony | min PHP |
|----------|---------|
| 5.4          | 8.0          |
| 6.4          | 8.1          |
| 7.0          | 8.2          |

---

Sure the CI runs through but the "Download dependencies"-step downloads for example with `sf_version: 7.0.*` and `php: 8.0` not sf 7.0 but sf 6.0.

---

Nice catch, thanks!